### PR TITLE
Update docs: NVIDIA drivers

### PR DESCRIPTION
**Pull request by alllle123**
Update documentation regarding installation and upgrade of new NVIDIA drivers for windows and linux.

Driver updated from Linux: 535.129.03 > 535.161.07 
Driver updated from Windows: 537.70 > 538.33
Added link to the compatibility list

### CODE REVIEW

**Review by tobias-urdin — APPROVED** [review]

**Review by tobias-urdin — CHANGES_REQUESTED** [review]
changes look good but there is a conflict that needs to be resolved for merge

File: compute/gpu-instances.rst [hunk 1]
```diff
@@ -107,7 +108,7 @@ Upgrading the driver
 --------------------
 From time to time, nVidia will release (and Binero will provide) and upgraded version of the GPU driver. This is in order to correct potential bugs and keep the software secure. When this happens, Binero strongly recommends (and in some cases, it will be required to maintain a working system) that the driver be updated on the instances running it. Please follow below instructions to upgrade the driver.
 
-The current version of the driver that we support is: Windows: **537.70** Linux **535.129.03**.
+The latest version of the driver that we support is ** Linux: 535.161.05 Windows: 538.33.**
```
Comment (tobias-urdin) [inline compute/gpu-instances.rst:111]: says 535.161.05 but the download says 535.161.07

File: compute/gpu-instances.rst [hunk 2]
```diff
@@ -45,10 +45,11 @@ A GPU instance :doc:`is launched <launching-an-instance/index>` the same way as
 
 Installing the driver
 ---------------------
+In order to use the GPU functionality, a driver from NVIDIA needs to be installed on the instance that has access to the virtual GPU. Please follow below instructions to install the driver.
 
-In order to use the GPU functionality, a driver from nVidia needs to be installed on the instance that has access to the virtual GPU. Please follow below instructions to install the driver.
-
-The current version of the driver that we support is: **535.129.03**.
+The current NVIDIA vGPU Software Version that we are running is: **16.4**
+The current latest driver we support is **Linux:535.161.05 Windows:538.33**
```
Comment (tobias-urdin) [inline compute/gpu-instances.rst:51]: says 535.161.05 but the download says 535.161.07

**Review by tobias-urdin — CHANGES_REQUESTED** [review]

File: compute/gpu-instances.rst (hunk 1 above)
Comment (alllle123) [reply compute/gpu-instances.rst]: Thanks, 

Have fixed and pushed new correct version.

File: compute/gpu-instances.rst (hunk 2 above)
Comment (alllle123) [reply compute/gpu-instances.rst]: Thanks, 

Have fixed and pushed new correct version.

**Review by tobias-urdin — APPROVED** [review]

**Pull request merged**